quarkus-opentelemetry-exporter-azure/deployment/src/main/java/io/quarkiverse/opentelemetry/exporter/azure/deployment/AzureExporterProcessor.java

```diff
@@ -64,9 +64,13 @@ void runtimeInitializedClasses(BuildProducer<RuntimeInitializedClassBuildItem> r
         runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem("io.netty.internal.tcnative.SSL"));
         runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem("io.netty.util.concurrent.GlobalEventExecutor"));
         runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem(
-                "com.azure.core.http.vertx.VertxAsyncHttpClientProvider$GlobalVertxHttpClient"));
+                "com.azure.core.http.vertx.VertxHttpClientProvider$GlobalVertxHttpClient"));
         runtimeInitializedClasses.produce(
-                new RuntimeInitializedClassBuildItem("com.azure.core.http.vertx.VertxAsyncHttpClientBuilder$DefaultVertx"));
+                new RuntimeInitializedClassBuildItem("com.azure.core.http.vertx.VertxHttpClientBuilder$DefaultVertx"));
+        runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem(
+                "com.azure.monitor.opentelemetry.autoconfigure.implementation.quickpulse.QuickPulseDataCollector"));
+        runtimeInitializedClasses.produce(new RuntimeInitializedClassBuildItem(
+                "com.azure.monitor.opentelemetry.autoconfigure.implementation.heartbeat.HeartbeatDefaultPayload"));
```

Comment:
What is the issue you observe without this runtime initialization?

Reply:
The `QuickPulseDataCollector` has multiple system mbeans defined as private static members. The native image compilation fails because of that.

Reply:
My question was related to 
`new RuntimeInitializedClassBuildItem("com.azure.monitor.opentelemetry.autoconfigure.implementation.heartbeat.HeartbeatDefaultPayload")`